release/models/interfaces/openconfig-if-aggregate.yang

```diff
@@ -204,12 +213,63 @@ module openconfig-if-aggregate {
       type leafref {
         path "/oc-if:interfaces/oc-if:interface/oc-if:name";
       }
+      status deprecated;
       description
         "Specify the logical aggregate interface to which
         this interface belongs";
     }
   }
 
+  grouping aggregation-ethernet-config {
+    description
+      "Adds configuration items for dedicated aggregation container
+      for Ethernet interfaces belonging to a logical aggregate / LAG";
+
+    leaf aggregate-id {
+      type leafref {
+        path "/oc-if:interfaces/oc-if:interface/oc-if:name";
+      }
+      description
+        "Specify the logical aggregate interface to which
+        this interface belongs";
+    }
+
+    leaf port-priority {
+      type uint16;
+      default "32768";
```

Comment:
Suggest dropping the default - this is per-implementation and not defined in the LACP specification